crates/sui-types/src/balance_change.rs

```diff
@@ -25,9 +25,11 @@ pub struct BalanceChange {
 fn coins(objects: &[Object]) -> impl Iterator<Item = (&SuiAddress, TypeTag, u64)> + '_ {
     objects.iter().filter_map(|object| {
         let address = match object.owner() {
-            Owner::AddressOwner(sui_address) | Owner::ObjectOwner(sui_address) => sui_address,
+            Owner::AddressOwner(sui_address)
+            | Owner::ObjectOwner(sui_address)
+            // TODO-DNS is this right?
+            | Owner::ConsensusAddressOwner { owner: sui_address, .. } => sui_address,
```

Comment:
Yes this is correct. I assume if we have a merge coins command that uses a mix of consensus_owned/owned objects this will still work properly?

Reply:
(but the answer should be yes)